modules/framework/galasa-parent/dev.galasa.framework.api.users/src/main/java/dev/galasa/framework/api/users/internal/routes/UsersDeleteRoute.java

```diff
@@ -0,0 +1,118 @@
+
+import dev.galasa.framework.spi.auth.IInternalAuthToken;
+
+import dev.galasa.framework.api.common.BaseRoute;
+import dev.galasa.framework.api.common.Environment;
+import dev.galasa.framework.api.common.EnvironmentVariables;
+import dev.galasa.framework.api.common.InternalServletException;
+import dev.galasa.framework.api.common.QueryParameters;
+import dev.galasa.framework.api.common.ResponseBuilder;
+import dev.galasa.framework.api.common.ServletError;
+import dev.galasa.framework.auth.spi.IAuthService;
+import dev.galasa.framework.spi.FrameworkException;
+import dev.galasa.framework.spi.auth.AuthStoreException;
+import dev.galasa.framework.spi.auth.IAuthStoreService;
+import dev.galasa.framework.spi.auth.IUser;
+
+public class UsersDeleteRoute extends BaseRoute{
+
+    // Regex to match endpoint /users/{someLoginId}
+    private static final String path = "\\/([a-zA-Z0-9\\-\\_]+)\\/?";
+
+    private Environment env;
+    private IAuthStoreService authStoreService;
+    private IAuthService authService;
+    private BeanTransformer beanTransformer;
+    private Pattern pathPattern;
+
+    public UsersDeleteRoute(ResponseBuilder responseBuilder, Environment env,
+            IAuthService authService) {
+        super(responseBuilder, path);
+        this.env = env;
+        this.authService = authService;
+        
+        String baseServletUrl = env.getenv(EnvironmentVariables.GALASA_EXTERNAL_API_URL);
+        
+        this.beanTransformer = new BeanTransformer(baseServletUrl);
+        this.authStoreService = authService.getAuthStoreService();
+        this.pathPattern = getPathRegex();
+    }
+
+    @Override
+    public HttpServletResponse handleDeleteRequest(
+        String pathInfo,
+        QueryParameters queryParams,
+        HttpServletRequest request,
+        HttpServletResponse response
+    ) throws FrameworkException{
+
+        logger.info("handleDeleteRequest() entered");
+
+        String userNumber = extractUserNumberFromUrl(request, pathInfo);
+        IUser user = authStoreService.getUser(userNumber);
+
+        deleteUser(user);
+
+        logger.info("handleDeleteRequest() exiting");
+        return getResponseBuilder().buildResponse(request, response, HttpServletResponse.SC_NO_CONTENT);
+    }
+
+    private String extractUserNumberFromUrl(HttpServletRequest request, String pathInfo) throws InternalServletException{
+
+        try{
+
+            Matcher matcher = pathPattern.matcher(pathInfo);
+            matcher.matches();
+
+            String userNumber = matcher.group(1);
+            return userNumber;
+
+        }catch(Exception ex){
+            ServletError error = new ServletError(GAL5085_FAILED_TO_GET_LOGIN_ID_FROM_URL);
+            throw new InternalServletException(error, HttpServletResponse.SC_NOT_FOUND, ex);
+        }
+
+    }
+
+    private void deleteUser(IUser user) throws AuthStoreException, InternalServletException{
+
+        try{
+
+            if(user == null){
```

Comment:
Nitpick - more formatting inconsistencies:
- The `try` is missing a space before the `{`
- The catch is missing a space after the `}`

You could also change the `if` to be formatted like:
```java
if (user == null) {
```